requirements/requirements-dev.txt

```diff
@@ -5,5 +5,5 @@ plotly~=5.1.0
 pytest~=6.2.4
 coverage~=5.5
 codecov~=2.1.12
-pdoc3~=0.9.2
+pdoc3~=0.10.0
```

Comment:
Are we sure that this won't add documentation for other variables which shouldn't be added?